rampsdk/src/main/java/network/ramp/sdk/events/model/Events.kt

```diff
@@ -9,6 +9,8 @@ sealed class Event(val type: EventType) {
     }
 }
 
+@JsonClass(generateAdapter = true)
+data class Close(var payload: WidgetClosePayload? = null) : Event(EventType.CLOSE) // TO_VERIFY Not in documentation but in received events
```

Comment:
Not in the documentation because it's "internal" for SDK only.
I wonder how this SDK closed the widget earlier without a handler for this event